src/pages/Rules/rules.css

```diff
@@ -0,0 +1,48 @@
+.card {
+  padding: 2rem 1.5rem;
+  margin: 2rem auto;
+  width: 500px;
```

Comment:
You can use rem here

Reply:
I'll change that, Thank you @harshalini for PR review